src/lesson_4/script.js

```diff
@@ -0,0 +1,97 @@
+let sum = 0.1 + 0.2;
```

Comment:
```suggestion
const sum = 0.1 + 0.2;
```
the value in unchanging through the code